src/providers/messages.js

```diff
@@ -30,7 +31,15 @@ function buildSubscribeMessages() {
         subscriptionMessages[objectType] = (subscriptionDetails) => {
             let message;
 
-            if (isEventOrAlarmType(objectType)) {
+            if (isEventType(objectType)) {
+                message = `{
+                    "type": "${dataType}",
+                    "id": "${subscriptionDetails.subscriptionId}",
+                    "options": {
+                        "instance": "${subscriptionDetails.instance}"
```

Comment:
removed "processor" from the subscribe message for events as yamcs was rejecting it.

Open network tab and navigate to 'Events' on our testing environment to see the constant disconnect/reconnect. This fixes it.

Reply:
Yikes! Good catch. This has worked recently (and still works in our sim environment) so I wonder if this is a change in Yamcs itself?

Reply:
I think this was due to a change in the Yamcs API. I referenced the events subscription API and noticed it did not contain the processor parameter.